README.md

```diff
@@ -68,7 +68,7 @@ keeping it up to date for you.
 
 To get involved with the project see our [CONTRIBUTING](CONTRIBUTING.md)
 documentation. Our [SIG Calls](CONTRIBUTING.md#join-a-sig-call) are Mondays at
```

Comment:
Could say "every second Monday"?

Reply:
Updated it to be every other monday if that works